src/rpc/mining.cpp

```diff
@@ -247,7 +247,10 @@ static UniValue prioritisetransaction(const JSONRPCRequest& request)
 
     LOCK(cs_main);
 
-    uint256 hash = ParseHashStr(request.params[0].get_str(), "txid");
+    uint256 hash;
+    if (!ParseHashStr(request.params[0].get_str(), hash)) {
+        throw JSONRPCError(RPC_INVALID_PARAMETER, "txid must be hexadecimal string (not '" + request.params[0].get_str() + "')");
```

Comment:
I know this is unchanged from the previous version, but could there be circumstances where this opens up for log file injection? Safer not to echo the input?